modules/admin_manual/pages/enterprise/authentication/kerberos.adoc

```diff
@@ -189,15 +189,14 @@ Library version => Kerberos 5 release 1.17
 ----
 --
 
-* Install `mod_auth_kerb` for Apache
+* Install handy command-line tools for Kerberos: +
+Note that `krb5-user` is not an actual requirement, but it will provide handy command-line tools for Kerberos, such as the ability to manipulate keytab files.
```

Comment:
It is an actual requirement, we need the `kvno` command which is part of that package (on Ubuntu). As usual the package might be named differently on other Linux flavours.

```suggestion
Note that `krb5-user` is an actual requirement, the Kerberos implementation in the WND app requires the `kvno` command which is contained in that package.
```